pkg/acsengine/Get-AzureConstants.ps1

```diff
@@ -115,6 +115,8 @@ Get-Locations() {
 	$locationList += "germanynortheast"
 	$locationList += "usgoviowa"
 	$locationList += "usgovvirginia"
+	$locationList += "usgovarizona"
```

Comment:
You can undo these changes, this file should actually be deleted, since we only use the python file.